changes/537.feature

```diff
@@ -0,0 +1 @@
+Check resource by root folder name of subdirectory and apply mount info of root to its mounted subdirectories.
```

Comment:
What do you mean by "check resource" (e.g., is it related with resource slots)?

Reply:
This refers to an already implemented function that checks whether the folder to be mounted exists or not. 
And this function also returns various information such as permission, host, path, etc.

So It is implemented carrying all subdirectories' various informations to `backend.ai-agent`, and `backend.ai-agent` check subdirectories' existence (if not exist, don't work and return error message). This feature is already implemented, too.